Tests/Docker/Runner/DataLoader/ProviderInitializerTest.php

```diff
@@ -206,9 +206,10 @@ public function testInitializeInputAbs()
             ],
             '/tmp/random/data'
         );
+        
         self::assertInstanceOf(InputFileLocal::class, $stagingFactory->getFileInputStrategy(InputStrategyFactory::LOCAL));
         self::assertInstanceOf(InputTableLocal::class, $stagingFactory->getTableInputStrategy(InputStrategyFactory::LOCAL, '', new InputTableStateList([])));
-        self::assertInstanceOf(InputFileABSWorkspace::class, $stagingFactory->getFileInputStrategy(InputStrategyFactory::WORKSPACE_ABS));
+        // self::assertInstanceOf(InputFileABSWorkspace::class, $stagingFactory->getFileInputStrategy(InputStrategyFactory::WORKSPACE_ABS));
```

Comment:
@odinuv not sure what to do about this.  it fails because in the new file absWorkspaceStrategy constructor it initializes the blob client https://github.com/keboola/input-mapping/blob/master/src/File/Strategy/ABSWorkspace.php#L41 which of course fails in this `dummy` scenario

Reply:
I usppose I could change the constructor in the IM lib so that it doesn't init the blob client there but uses an initBlobClient method like we have elsewhere with the sapiClient, but 🤷‍♂️

Reply:
oh, i didn't realize that `BlobRestProxy::createBlobService` communicates with something, the blobl client should be lazily initialized in IM then.

Reply:
yeah, we agreed about that in the standup, here's the PR https://github.com/keboola/input-mapping/pull/86